### PR TITLE
Various updates and a fix for issue #15

### PR DESCRIPTION
**Pull request by nabil-boag**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -29,34 +35,50 @@ module.exports = function (grunt) {
...
   grunt.initConfig({
-    pkg:  grunt.file.readJSON("package.json"),
-    env: grunt.option('env') || 'dev',
-
     app: {
       name: 'angular-multimocks',
       source_dir: 'app/src',
       build_dir: 'app/build',
-      package_dir: 'app/package'
+      demo_dir: '<%= app.build_dir %>/demo',
```
Comment (afternoon) [inline Gruntfile.js]: Remove pls.

File: Gruntfile.js [hunk 2]
```diff
@@ -29,34 +35,50 @@ module.exports = function (grunt) {
...
     clean: {
-      build : '<%= app.build_dir %>',
-      package : '<%= app.package_dir %>'
+      build: '<%= app.build_dir %>',
+      package: '<%= app.package_dir %>'
     },
 
     jshint: {
       source: [
+        '*.js',
```
Comment (afternoon) [inline Gruntfile.js]: Share this list of globs between jshint and jscs tasks?

File: Gruntfile.js [hunk 3]
```diff
@@ -118,10 +140,31 @@ module.exports = function (grunt) {
...
+      }
+    },
+
     watch: {
       dev: {
         files: ['<%= app.source_dir %>/**/*'],
-        tasks: ['build', 'test:dev']
+        tasks: ['build', 'test:unit'],
+        options: {
+          livereload: true
```
Comment (afternoon) [inline Gruntfile.js:162]: Did you test livereload?

File: README.md [hunk 4]
```diff
@@ -17,6 +15,15 @@ You can use Angular Multimocks to quickly test your app works in all situations
 while developing or to provide mock data for a suite of automated acceptance
 tests.
 
+Demo
+----
+<http://tech.wonga.com/angular-multimocks/>
+
+
+Travis Status
+-------------
```
Comment (afternoon) [inline README.md]: Travis badge should be at top of README

File: Gruntfile.js (hunk 3 above)
Comment (nabil-boag) [inline Gruntfile.js:162]: Fixed by 060d69ad7647982f78d0899d48e3c76b336cb553

**Pull request merged**